tests/resources/functions/go/latest/tests.go

```diff
@@ -164,7 +165,7 @@ When you can have two!
 
 		if errResponse != nil {
 			Context.Error(errResponse)
-			return Context.Res.Text("", 500, nil)
+			return Context.Res.Text("", Context.Res.WithStatusCode(500))
```

Comment:
`WithStatusCode`? When did we decide to name it this way?

Reply:
Yes, Torsten approved. It's pattern used in Go, that any optional parameter is added by `withX` method. 

Example:
```
NewClient(WithAuth("my-authorization-key"))
```
> https://petomalina.medium.com/dealing-with-optional-parameters-in-go-9780f9bfbd1d

Reply:
Just `WithStatusCode()`: not idea, could conflict with a HTTP client library
Just `Context. WithStatusCode()`: Indicates it might work with request, or logger.
Exactly `Context.Res.WithStatusCode()`: Makes it clear it's for responses

--

Just `WithStatus()` - weird
Just `WithCode()` - weird
Exactly `WithStatusCode()` - Follows HTTP protocol naming convention